wp-seo-main.php

```diff
@@ -375,12 +375,11 @@ function wpseo_admin_init() {
 		}
 	}
 
-	if ( in_array( $pagenow, array( 'edit-tags.php' ) ) ) {
+	if ( $pagenow === 'edit-tags.php' ) {
 		$GLOBALS['wpseo_taxonomy'] = new WPSEO_Taxonomy;
 	}
 
-	if ( in_array( $pagenow, array( 'admin.php' ) ) ) {
-		// @todo [JRF => whomever] Can we load this more selectively ? like only when $_GET['page'] is one of ours ?
+	if ( 'admin.php' === $pagenow && strpos( (string) filter_input( INPUT_GET, 'page' ), 'wpseo' ) === 0 ) {
```

Comment:
Unfortunately we've already run into the fact that the filter extension is not always enabled, so this code cannot be used.